profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/compute/ProfitBricksComputeServiceLiveTest.java

```diff
@@ -113,4 +119,28 @@ public void testCreateNodeWithCustomHardwareUsingMins() throws Exception {
          client.destroyNodesMatching(inGroup(group + "custom"));
       }
    }
+
+   @Test
+   public void testCreateNodeWithInboundPorts() throws Exception {
+      pbApi = client.getContext().unwrapApi(ProfitBricksApi.class);
+
+      Template template = client.templateBuilder()
+              .options(TemplateOptions.Builder.inboundPorts(80, 22, 443)).build();
+      try {
+         int matches = 0;
+         client.getSecurityGroupExtension();
+         NodeMetadata node = getOnlyElement(client.createNodesInGroup(group + "inbound", 1, template));
+         DataCenterAndId datacenterAndId = DataCenterAndId.fromSlashEncoded(node.getId());
+         ProfitBricksApi pbApi = client.getContext().unwrapApi(ProfitBricksApi.class);
+         Server server = pbApi.serverApi().getServer(datacenterAndId.getDataCenter(), datacenterAndId.getId(), new DepthOptions().depth(5));
+         for (FirewallRule rule : server.entities().nics().items().get(0).entities().firewallrules().items()) {
+            if (rule.properties().portRangeStart() == 80 || rule.properties().portRangeStart() == 22 || rule.properties().portRangeStart() == 443) {
+               matches++;
+            }
+         }
+         Assert.assertEquals(3, matches);
+      } finally {
+         client.destroyNodesMatching(inGroup(group + "inbound"));
+      }
+   }
```

Comment:
Do you mean to call the super `testCreateAndRunAService` or the `createAndRunAServiceInGroup`, if i override the `createAndRunAServiceInGroup` i still have to create an additional test since its not a test its just a method?